### ISSUE TITLE
Insufficient text color contrast ratio

### ISSUE THREAD

**Issue by mueller-ma** [openhab/openhab-android#3067]
`TextViews` using `android:textColor="@color/empty_list_text_color"` have an insufficient text color contrast ratio:

> The item's text contrast ratio is 2.07. This ratio is based on an estimated foreground color of #B2B2B2 and an estimated background color of #FFFBFE. Consider increasing this item's text contrast ratio to 4.50 or greater. 

This color can be replaced by e.g. `?colorOnSurfaceVariant`.
